src/clojure_lsp/db.clj

```diff
@@ -11,34 +11,34 @@
 (defonce diagnostics-chan (async/chan 1))
 (defonce edits-chan (async/chan 1))
 
-(def version 5)
+(def version 1)
 
 (defn make-spec [project-root]
-  (let [lsp-db (io/file (str project-root) ".lsp" (str "sqlite." version ".db"))]
+  (let [lsp-db (io/file (str project-root) ".lsp" "sqlite.db")]
```

Comment:
So we will always use the same version of the db? what's the tradeoff now of that?

Reply:
The `version` is already considered in selecting from db so there's no reason to leave files hanging around